vmdb/db/migrate/20100308005125_create_miq_cim_instances.rb

```diff
@@ -14,7 +14,7 @@ def self.up
       t.column :agent_id,         :integer
       t.column :stat_id,          :integer
       t.column :stat_top_id,        :integer
-      t.timestamps
+      t.timestamps :null => true
```

Comment:
Yeah, good point... I don't know that any new code will rely upon the old way.  As long as we clean up all of the warnings on existing code now, we should be ok when the default changes.